focus_areas/templates/question_template.md

```diff
@@ -1,37 +1,21 @@
-# [Area of Analysis]: [Focal Concern]
+## Description
```

Comment:
I think we wanted to keep the first four lines of the old template to also have the Title and Question in the new template.

Please add Title and Question back into the template.